lineQueueClient/src/Signup/Signup.js

```diff
@@ -1,17 +1,85 @@
-import React from 'react'
-import {
-  Text,
-  View
-} from 'react-native';
+import React, {Component} from 'react'
+import {Text, View, TextInput, StyleSheet, Image, Button} from 'react-native';
 
-const Signup = () => {
-	return (
-		<View>
-			<Text>
-				This is the signup page
-			</Text>
-		</View>
-	)
+export default class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      text: ''
+    };
+  }
+
+  render() {
+    return (
+      <View>
+        <View style={styles.containerBlack}>
```

Comment:
Header container should be extracted to different component for reusability in other screens. i.e `export default class Header extends Component` in a different directory